payments/models.py

```diff
@@ -105,7 +124,12 @@ class Rate(models.Model):
     price = models.DecimalField(max_digits=6, decimal_places=2,
                                 verbose_name='price (USD)')
     capacity = models.PositiveIntegerField()
-
+    discount = models.BooleanField(default=False)
+    discount_code = models.SlugField(help_text='This will be the code given to'
+                                     'a customer receiving a discount in the '
+                                     'form of https://workshops.qiime.org/wor'
+                                     'kshop_slug/rate=discount_code',
+                                     default=uuid.uuid4)
```

Comment:
When adding more than one discount rate, all the rates have the same UUID value:

![image](https://cloud.githubusercontent.com/assets/274668/18527961/cbdbd40a-7a7b-11e6-95d3-26043ed5fb3a.png)